Dockerfile

```diff
@@ -26,6 +26,7 @@ EXPOSE 8000
 ENV LONGHORN_MANAGER_IP http://localhost:9500
 ENV LONGHORN_UI_PORT 8000
 
-USER nginx
+RUN usermod -u 1000 nginx && chown -R nginx:nginx /var/log/nginx
```

Comment:
`chown -R nginx:nginx /var/log/nginx` seems not right, because it will possibly leave some other permission leftover belonging to the original nginx uid.

Reply:
Removed, only change `User 499`